apps/web/app/features/stacking/components/preset-pools.tsx

```diff
@@ -0,0 +1,140 @@
+// import { IconEdit } from '@tabler/icons-react';
+import { MIN_DELEGATED_STACKING_AMOUNT_USTX } from '~/constants/app';
+
+import {
+  NetworkInstance,
+  NetworkInstanceToPoxContractMap,
+  PayoutMethod,
+  PoolName,
+  PoxContractName,
+} from '../utils/types-preset-pools';
+import { PoolIcon } from './pool-icon';
+
+export const pools = {
+  'FAST Pool': {
+    name: PoolName.FastPool,
+    description:
+      'Enjoy automatic pool operations.' +
+      ' ' +
+      'You can increase the locking amount for the next cycle.' +
+      ' ' +
+      'Locked STX will unlock 1 day after the end of the cycle.',
+    duration: 1,
+    website: 'https://fastpool.org',
+    payoutMethod: PayoutMethod.STX,
+    poolAddress: {
+      [NetworkInstance.mainnet]:
+        NetworkInstanceToPoxContractMap[NetworkInstance.mainnet][PoxContractName.WrapperFastPool],
+      [NetworkInstance.testnet]:
+        NetworkInstanceToPoxContractMap[NetworkInstance.testnet][PoxContractName.WrapperFastPool],
+      [NetworkInstance.devnet]:
+        NetworkInstanceToPoxContractMap[NetworkInstance.devnet][PoxContractName.WrapperFastPool],
+    }, // pool address is the same as pool contract
+    poxContract: PoxContractName.WrapperFastPool,
+    minimumDelegationAmount: 40_000_000,
+    icon: <PoolIcon src="/icons/32x32_FastPool.png" />,
+    allowCustomRewardAddress: false,
+    disabled: false,
+  },
+
+  PlanBetter: {
```

Comment:
@alter-eggo 

@friedger added `Fast Pool v2` in this PR here - can you also add that here?

Reply:
will add once that [pr](https://github.com/leather-io/earn/pull/231) is merged